web_src/css/shared/settingstable.css

```diff
@@ -0,0 +1,24 @@
+/* This css is for table elements inside all levels settings pages */
+.g-grid-setting-right .table.segment {
+  padding: 0;
+  font-size: 0.9em;
+  overflow-x: auto;
+}
+
+.g-grid-setting-right .table.segment:not(.striped) thead th:last-child {
+  padding-right: 5px !important;
+}
+
+.g-grid-setting-right .table.segment th {
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.g-grid-setting-right .table.segment:not(.select) th:first-of-type,
+.g-grid-setting-right .table.segment:not(.select) td:first-of-type {
+  padding-left: 15px !important;
+}
+
+.g-grid-setting-right .table.segment form tbody button[type="submit"] {
+  padding: 5px 8px;
```

Comment:
Why this? There are also other `<button>` elements in forms

Reply:
This is copied from the old code, which is table related.

Reply:
I am pretty sure that we do not need them ... will do more fine tunes